src/meshapi/tests/test_install.py

```diff
@@ -103,6 +103,14 @@ def test_update_install_unset_install_number(self):
         self.assertEqual(install.id, uuid.UUID("23ef170c-f37d-44e3-aaac-93dae636c86e"))
         self.assertEqual(install.install_number, 45)
 
+    def test_osticket_number_leading_zeros(self):
```

Comment:
Can/should we add a testcase to reproduce the issue?

Reply:
That's what this is, right? Like the current issue is that leading zeros aren't tracked. And this test confirms that they are